src/main/java/com/cloudbees/jenkins/plugins/docker_build_env/Docker.java

```diff
@@ -110,9 +112,11 @@ public boolean pullImage(String image) throws IOException, InterruptedException
 
 
     public String buildImage(FilePath workspace, String dockerfile, boolean forcePull, boolean noCache) throws IOException, InterruptedException {
+        File tempImageIdFile = File.createTempFile("dcbep-", ".iid");
```

Comment:
should be FilePath tempImageIdFile = workspace.createTempFile("dcbep-", ".iid");